### ISSUE TITLE
Screen capturing not working

### ISSUE THREAD

**Issue by Aybee5** [gioxx/MarvellousSuspender#106]
Please complete the following information when submitting a feature request or bug report.
 * Extension version:
 v7.1.6.2
 
 * Browser name & version:
 Brave Version 1.23.75 Chromium: 90.0.4430.93 (Official Build) (64-bit)
 
 * Operating system & version:
 Ubuntu 20.04

When I attempt to enable screen capturing on the settings, I get this error in the console
![image](https://user-images.githubusercontent.com/42099896/117697943-ff49eb80-b1ba-11eb-8e1c-4fe213163738.png)

**Comment by bitnom** [comment]
Confirmed by me. Same extension version and browser. It's not possible to activate screenshots once disabled. I only fiddled with it to begin with because I noticed tab previews of suspended tabs are not working.

**Comment by PorridgePi** [comment]
Yes, this is confirmed to be a bug in [#81](https://github.com/gioxx/MarvellousSuspender/issues/81). You can fix the problem by turning on the option "Allow access to file URLs".

**Comment by gioxx** [comment]
Duplicate #81, work around available, already fixed on GitHub (but if you want the Chrome Store version, you have to wait TMS 7.1.6.3, we are working on it to publish soon).
